content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: "Exercise 3.0 - Using Ansible to Implement Security"
+workshops: ansible_tower
+workshop_weight: 30
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Exercise 3.0 - Using Ansible to Implement Security
+
```

Comment:
The STIG content is coming.  I just wanted to get this much added, to get started.